### PR TITLE
Add color to StreamR

### PR DESCRIPTION
**Pull request by jkotalik**
Made a PR to the wrong place.

The only thing I'm concerned about is the performance. We are suddenly sending significantly more bytes per character. Any ideas on how to reduce it are appreciated.

### CODE REVIEW

File: Startup.cs [hunk 1]
```diff
@@ -47,16 +47,23 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment environment)
...
-            app.UseSignalR(route =>
+            app.UseEndpoints(endpoints =>
             {
-                route.MapHub<StreamHub>("/stream");
+                  // Mapping of endpoints goes here:
+                endpoints.MapControllers();
+                endpoints.MapRazorPages();
+                endpoints.MapHub<StreamHub>("/stream", o =>
+                {
+                    //o.TransportMaxBufferSize = 1000000;
```
Comment (davidfowl) [inline Startup.cs:63]: Since you cleaned this up. Remove these.

File: StreamR.csproj [hunk 2]
```diff
@@ -7,7 +7,6 @@
 
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="3.0.0-preview-19075-0444" />
     <PackageReference Include="Microsoft.AspNetCore.SignalR" Version="3.0.0-preview-19075-0444" />
```
Comment (davidfowl) [inline StreamR.csproj:10]: Remove this too? When was this project made?

Comment (BrennanConroy) [reply StreamR.csproj:10]: A long time ago :)

**Comment by BrennanConroy** [comment]
>We are suddenly sending significantly more bytes per character. Any ideas on how to reduce it are appreciated.

Yeah, that was my main concern with adding color. We should come up with a custom compacting format to encode the character + color.

Also, we can switch to MessagePack instead of Json

**Comment by jkotalik** [comment]
I'm going to close this for now and revive it later (when I actually have free time).

**Pull request closed without merging**